docs/source/index.rst

```diff
@@ -96,3 +96,9 @@ Indices and tables
 .. |license| image:: https://img.shields.io/github/license/indico/indico.png
                    :alt: License
                    :target: https://github.com/indico/indico/blob/master/COPYING
+
+Contact
+++++++++++++++++++
```

Comment:
that's way too many `+` chars... the underline should be just as long as the line before